textile.go

```diff
@@ -258,9 +258,9 @@ func (x *initCmd) Execute(args []string) error {
 		LogToDisk:       !x.Logs.NoFiles,
 		Debug:           x.Logs.Debug,
 		CafeOpen:        x.CafeOptions.Open,
-		CafePublicIP:    x.CafeOptions.PublicIP,
-		CafeURL:         x.CafeOptions.URL,
-		CafeNeighborURL: x.CafeOptions.NeighborURL,
+		CafePublicIP:    envOrFlag("CAFE_HOST_PUBLIC_IP", x.CafeOptions.PublicIP),
```

Comment:
Allows these options to be passed as ENV vars, which is needed to get through docker run.